API2.md

```diff
@@ -1,24 +1,48 @@
 # Journalist API v2
 
-This package (in `securedrop/journalist_app/api2`) implements and documents the synchronization strategy for the v2
-Journalist API.
-
-| File                                  | Contents                             |
-| ------------------------------------- | ------------------------------------ |
-| `README.md`                           | Specification                        |
-| `__init__.py`                         | Server implementation                |
-| `../../tests/test_journalist_api2.py` | Test suite for server implementation |
+The `securedrop.journalist_app.api2` package implements the synchronization
+strategy for the v2 Journalist API.
+
+| File/module                             | Contents                                                                                                       |
+| --------------------------------------- | -------------------------------------------------------------------------------------------------------------- |
+| `API2.md` (you are here)                | Specification                                                                                                  |
+| `securedrop.journalist_app.api2`        | Flask blueprint for `/api/v2/`                                                                                 |
+| `securedrop.journalist_app.api2.events` | Event-handling framework                                                                                       |
+| `securedrop.journalist_app.api2.shared` | Helper functions factored out of and still shared with the v1 Journalist API (`securedrop.journalist_app.api`) |
+| `securedrop.journalist_app.api2.types`  | Types                                                                                                          |
+| `securedrop.tests.test_journalist_api2` | Test suite for server implementation                                                                           |
 
 A client-side implementation should be able to interact with the endpoints
-implemented in `__init__.py` according to this specification.
+implemented in `securedrop.journalist_app.api2` according to this specification.
+
+## Goals and properties
```

Comment:
@vickiniu, this section is inspired by comments of yours in the onboarding session you gave last week.  Let me know if I've missed anything you think is important to document about the design motivations we've followed in developing the v2 Journalist API!